packages/backend/src/graphql/invoice/invoiceInput.ts

```diff
@@ -0,0 +1,11 @@
+import { builder } from '../builder'
+
+export const InvoiceInput = builder.inputType('InvoiceInput', {
+  fields: (t) => ({
+    customerAddress: t.string({ required: false }),
+    customerName: t.string(),
+    invoiceDate: t.field({ type: 'Date' }),
+    organizationId: t.id(),
+    sendDate: t.field({ type: 'Date', required: false }),
```

Comment:
what about payDate?
same question about invoiceInputValidations.ts and invoiceCreateMutationField.ts

Reply:
This will be included in another ticket.